### DISCUSSION TITLE
toogle table view

### DISCUSSION THREAD

**Discussion by toog** [fbaligand/kibana-enhanced-table#282] [Ideas]
Hello 

it would be useful in some cases to rotate the table to have the column names in row and the values in column 2
like the advanced rendering of the discovers when clicking on a document

|  field1 | field2  | field3  | ...   | fieldN  |
|---|---|---|---|---|
|  1 | toto  | mydata  | ..  | 0.002  |
|  2 | tutu  | yourdata  |  .. | 0.369  |
|  3 | titi  |  data |  .. | 0.5478 |

to 

|  fields | values  |
|---|---|
|  field1<br> field2 <br> field3 <br> ... <br> fieldN     | 1 <br> toto <br>mydata  <br> .. <br> 0.002 | 
|  field1<br> field2 <br> field3 <br> ... <br> fieldN     | 2 <br> tutu<br> yourdata  <br> .. <br> 0.369| 
|  field1<br> field2 <br> field3 <br> ... <br> fieldN     | 3 <br> titi <br> data  <br> .. <br> 0.5478| 
        
there is some function to do that or futur functionnality ?

**Comment by fbaligand** [comment]
Hi @toog,

I'm sorry but this is actually not a feature. And at this day, this is not in the roadmap.

I guess that you want that in Document Table viz (I mean not Enhanced Table viz), in the same way of Discover?
